packages/vuetify/src/components/VTooltip/VTooltip.sass

```diff
@@ -17,6 +17,9 @@
     width: auto
     opacity: 1
     pointer-events: none
+    
+    &[class*="--fixed"]
```

Comment:
```suggestion
    &[class*="-fixed"]
```

Reply:
why?

Reply:
both are ok, but [class*="-fixed"] is consistent with the siblings

Reply:
And `-` version is not consistent with other components, for example https://github.com/vuetifyjs/vuetify/blob/fae3f271f484e84169b0563a07682f76182b0e2f/packages/vuetify/src/components/VMenu/VMenu.sass#L28

Reply:
agreed